keel-api/src/test/resources/examples/titus-cluster-with-artifact-example.yml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: titus.spinnaker.netflix.com/v1
+kind: cluster
+metadata:
+  serviceAccount: my-email@spinnaker.io
+spec:
+  moniker:
+    app: keeldemo
+    stack: examples
+    detail: ec2v1
+  container:
+    organization: emburns
+    image: spin-titus-demo
+    tagVersionStrategy: semver_tag
```

Comment:
oh god i'm getting pedantic again i'm sorry — can we kebab-case-ify this? feels like implementation details bleeding out into our spec format

Reply:
ugh erik. I can try.

Reply:
Ok I fixed it. That was much less painful than I thought it would be

Reply:
so glad I dropped by 💅